pkg/jwt/handler.go

```diff
@@ -30,9 +30,10 @@ type Handler struct {
 // Reply will be of the form {"token": "<TOKEN>"}.
 func (j *Handler) Login(config config.Credentials) http.HandlerFunc {
 	return func(w http.ResponseWriter, r *http.Request) {
-		span := opentracing.FromContext(r.Context(), "token.extract")
+		_, span := trace.StartSpan(r.Context(), "token.extract")
```

Comment:
We can get rid of the trace, it's a usual CPU bound operation, we are more concerned about network calls i.e IO bound

Reply:
what we could actually do he needed is add annotate the existing tracing from the context with such information, however given it's request throughput is very low, I don't think it worth in the end

Reply:
Just for the record this trace was added in #270.  
I agree with the trace not being too useful, the metrics achieved the intended purpose of the PR; let's remove the trace.

Reply:
it was part of teh auth investigation that is finished and it was not Janus issue at all but the calling-side problems